### PR TITLE
set npm package.json

### PR DESCRIPTION
**Pull request by xtine**
## Set NPM details
- Set `package.json` details (name, version) so that `fec-cms` can be initialized as a local NPM package for `fec-pattern-library` to access assets.
- Remove `all.scss` because pattern library will be able access scss files to compile.

## Impacted areas of the application
Does not impact any part of the working application. Setting up `fec-cms` as an npm package only sets details for local usage.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+  "name": "fec-cms",
+  "version": "1.0.0",
```
Comment (ccostino) [inline package.json]: Should we set the `license` and `repository` too while we're at it just to get rid of the warnings?

Comment (xtine) [reply package.json]: What `license` should we use? I just added the repo info.

Comment (ccostino) [reply package.json]: 🤔 really good question, I'm not sure!  We'll have to ask around and see what would be appropriate.

Comment (xtine) [reply package.json]: Just checked, `fec-style` doesn't have a license and it's a published package... 😆

Comment (ccostino) [reply package.json]: Whoops, that probably shouldn't have been the case, d'oh!

Comment (xtine) [reply package.json]: Okay, we have quickly discovered https://github.com/18F/open-source-policy/blob/master/policy.md#open-source-licenses, which directs us to use `CC0`. It's already defined on the bottom of our `LICENSE.md`, but I'm explicitly including in the license field here as well.

**Pull request merged**